docs/TempUG.md

```diff
@@ -304,7 +306,7 @@ Format: `addPerson -name NAME [-phone PHONE_NUMBER] [-email EMAIL] [-address ADD
 Example usages:
 * `addPerson -name John`
 * `addPerson -name John -phone 91234567 -email test@gmail.com -address 10 Kent Ridge Drive -subject MATHEMATICS`
-* In `STUDENTS` list :
+* In ___STUDENTS list___ :
```

Comment:
not sure do we need to standardize this with the "#### schedule list" in line 334 ?

Reply:
I am not sure too, can askk the rest what they think ?

Reply:
Sure, we can bring this during the meeting